pkgs/by-name/la/lan-mouse/package.nix

```diff
@@ -1,32 +1,33 @@
-{ stdenv
-, rustPlatform
-, fetchFromGitHub
-, lib
-, darwin
-, glib
-, gtk4
-, libadwaita
-, libX11
-, libXtst
-, pkg-config
-, wrapGAppsHook4
+{
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  lib,
+  glib,
+  gtk4,
+  libadwaita,
+  libX11,
+  libXtst,
+  pkg-config,
+  wrapGAppsHook4,
+  git,
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "lan-mouse";
-  version = "0.9.1";
+  version = "0.10.0";
 
   src = fetchFromGitHub {
     owner = "feschber";
     repo = "lan-mouse";
     rev = "v${version}";
-    hash = "sha256-BadpYZnZJcifhe916/X+OGvTQ4FQeTLnoy0gP/i5cLA=";
+    hash = "sha256-ofiNgJbmf35pfRvZB3ZmMkCJuM7yYgNL+Dd5mZZqyNk=";
   };
 
   nativeBuildInputs = [
-    glib # needed in both {b,nativeB}uildInptus
     pkg-config
     wrapGAppsHook4
+    git
```

Comment:
They use `git` to obtain the latest tag for debugging. Can you patch it to re-use `version` instead? I have that coded here:

https://github.com/chaotic-cx/nyx/blob/376a2e022a5d8fa21cecb5bb0fef0cb54db5cdfc/pkgs/lan-mouse-git/default.nix#L18-22